DotnetRuntimeBootstrapper/CreateBootstrapperTask.cs

```diff
@@ -19,10 +20,20 @@ public class CreateBootstrapperTask : Task
     [Required]
     public string TargetFilePath { get; set; } = default!;
 
+    public string MicrosoftNETCoreApp { get; set; } = default!;
+
+    public string MicrosoftWindowsDesktopApp { get; set; } = default!;
+
+    public string MicrosoftAspNetCoreApp { get; set; } = default!;
+
+    public string MicrosoftWebView2Runtime { get; set; } = default!;
+
```

Comment:
What about a few properties (in the form of a string array that can be the form of an msbuild item array) that allows 3rd party or workload runtimes to be included as well (and even a way for bootstrapper to download them but with a notification message at least letting them know the be careful about it) Alternatively the url that the task would write to the runtimeconfig.json file *could* be encrypted + base64 encoded to avoid easy change from a bad actor.

Reply:
@AraHaan I think that you would need more options per item (runtime).
You need:
- Display name
- Version
- URL (from where to download the installer)
- Setup parameters (we want silent install
- A way to check if a specific item is installed.

The first 3 items are easy to do (we can agree on a separator and I can join them to form a single string), but how you will be able to configure how to check if specific runtime is already installed? Sometimes you must check the registry, sometimes check if a specific folder exists, etc.

Encrypting is a good idea, but I would suggest calculating checksum and storing it in resources in the host app (we already are editing resources, so it shouldn't be that hard)